commands/publish/command.js

```diff
@@ -110,6 +110,11 @@ exports.builder = (yargs) => {
       hidden: true,
       type: "boolean",
     },
+    "summary-file": {
+      // Json output.
+      hidden: true,
```

Comment:
Why should this be a hidden option?

Reply:
honestly been awhile since I looked at this, so open to suggestions